roles/upgrade/pre-upgrade/defaults/main.yml

```diff
@@ -6,6 +6,7 @@ drain_nodes: true
 drain_retries: 3
 drain_retry_delay_seconds: 10
 
+upgrade_node_always_cordon: true
```

Comment:
weird to set it as true by default, isn't it ?

Reply:
Doh, fixed.